.rubocop.yml

```diff
@@ -1,4 +1,6 @@
-require: rubocop-rspec
+require: 
+  - rubocop-rails
+  - rubocop-rspec
```

Comment:
Rails cops were removed from RuboCop in [version 0.72](https://github.com/rubocop-hq/rubocop/releases/tag/v0.72.0).